config/default.yml

```diff
@@ -244,6 +247,10 @@ Rails/Blank:
 
 Rails/BulkChangeTable:
   Description: 'Check whether alter queries are combinable.'
+  Reference:
+    - https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table
+    - https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html
+    - https://github.com/rails/rails/pull/31331
```

Comment:
This is unnecessary as it is an internal comment to the implementation rather than published documentation.

Reply:
Thanks for the review. I updated this PR.

Reply:
I thought we may leave it in the reference, but it may not be necessary. I removed it.